src/vm/runners/builtin_runner/signature.rs

```diff
@@ -61,33 +62,48 @@ impl BuiltinRunner for SignatureBuiltinRunner {
             move |memory: &Memory,
                   address: &MaybeRelocatable|
                   -> Result<Vec<MaybeRelocatable>, MemoryError> {
-                let pubkey: Relocatable;
-                let msg;
+                let address = if let MaybeRelocatable::Int(_address) = address {
+                    return Err(MemoryError::AddressNotRelocatable);
+                } else if let MaybeRelocatable::RelocatableValue(address) = address {
+                    address
+                } else {
+                    unreachable!()
+                };
```

Comment:
Nit:
```suggestion
                let address = match address {
                    MaybeRelocatable::RelocatableValue(address) => address,
                    _ => return Err(MemoryError::AddressNotRelocatable),
                };
```
There's only two types of `MaybeRelocatable` so the catch-all is correct.

Reply:
Alternatively, make the second case the `else`.